recipes/mpitrampoline/meta.yaml

```diff
@@ -0,0 +1,74 @@
+{% set name = "MPItrampoline" %}
+{% set version = "5.5.0" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://github.com/eschnett/{{ name }}/archive/refs/tags/v{{ version }}.tar.gz
+  sha256: bdfb145b754cf13d38485808f79a5638d954dbe181e7bb53a1950c71deed6989
+
+build:
+  # Fortran compiler is not found on Windows
+  skip: true  # [win]
+  number: 0
+
+requirements:
+  build:
+    - cmake
+    - ninja
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - {{ compiler('fortran') }}
+    - {{ stdlib('c') }}
+  host:
+    - pip
+    - python
+    - setuptools
+  run:
+    - python
+
+test:
+  commands:
+    - test -e $PREFIX/include/mpi.h  # [unix]
+    - test -e $PREFIX/include/mpif.h  # [unix]
+    - test -e $PREFIX/include/mpi.mod  # [unix]
+    - test -e $PREFIX/include/mpi_f08.mod  # [unix]
+    - test -e $PREFIX/lib/libmpitrampoline${SHLIB_EXT}  # [unix]
+    - test -e $PREFIX/lib/pkgconfig/MPItrampoline.pc  # [unix]
+    - test -e $PREFIX/lib/pkgconfig/mpi-c.pc  # [unix]
+    - test -e $PREFIX/lib/pkgconfig/mpi-cxx.pc  # [unix]
+    - test -e $PREFIX/lib/pkgconfig/mpi-fort.pc  # [unix]
+    - test -e $PREFIX/lib/pkgconfig/mpi-c.pc  # [unix]
+    - test -e $PREFIX/bin/mpicc  # [unix]
+    - test -e $PREFIX/bin/mpicxx  # [unix]
+    - test -e $PREFIX/bin/mpifc  # [unix]
+    - test -e $PREFIX/bin/mpifort  # [unix]
+    - test -e $PREFIX/bin/mpiexec  # [unix]
+    - if not exist %LIBRARY_INC%\mpi.h exit 1  # [win]
+    - if not exist %LIBRARY_INC%\mpif.h exit 1  # [win]
+    - if not exist %LIBRARY_INC%\mpi.mod exit 1  # [win]
+    - if not exist %LIBRARY_INC%\mpi_f08.mod exit 1  # [win]
+    - if not exist %LIBRARY_BIN%\libmpitrampoline.dll exit 1  # [win]
+    - if not exist %LIBRARY_LIB%\pkgconfig\MPItrampoline.pc exit 1  # [win]
+    - if not exist %LIBRARY_LIB%\pkgconfig/mpi-c.pc  # [win]
+    - if not exist %LIBRARY_LIB%\pkgconfig/mpi-cxx.pc  # [win]
+    - if not exist %LIBRARY_LIB%\pkgconfig/mpi-fort.pc  # [win]
+    - if not exist %LIBRARY_LIB%\pkgconfig/mpi-c.pc  # [win]
+    - if not exist %LIBRARY_BIN%\mpicc exit 1  # [win]
+    - if not exist %LIBRARY_BIN%\mpicxx exit 1  # [win]
+    - if not exist %LIBRARY_BIN%\mpifc exit 1  # [win]
+    - if not exist %LIBRARY_BIN%\mpifort exit 1  # [win]
+    - if not exist %LIBRARY_BIN%\mpiexec exit 1  # [win]
```

Comment:
These scripts probably contain broken paths. Only the host PREFIX paths are automatically replaced with placeholders and repaired on the end user's machine. My understanding is that these scripts contain wrappers around compilers which are located in BUILD_PREFIX, so they are not being patched. You will need to manually patch these prefixes to be "/opt/anaconda1anaconda2anaconda3" as described in the docs [1]. Or you can replace them with unprefixed compiler names (like "$GCC" which our compiler packages set). Implementing compiler packages is tricky because you need to support both single prefix (user environment) and split prefix (host and build prefix for conda-build). I think the way this is handled is used activation scripts typically. I would suggest looking at our compiler package feedstocks for ideas on how to do it correctly or to see which environment variables are set by our compiler packages and use those.

If you do a test compile in the test section it may expose some of these issues.

[1] https://docs.conda.io/projects/conda-build/en/stable/resources/make-relocatable.html
[2] https://docs.conda.io/projects/conda-build/en/stable/resources/activate-scripts.html